internal/controller/install/common_helpers.go

```diff
@@ -371,6 +380,36 @@ func createEnv(crdEnv []corev1.EnvVar) []corev1.EnvVar {
 	return envVars
 }
 
+func CreateProbesWithScheme(scheme corev1.URIScheme) (*corev1.Probe, *corev1.Probe) {
```

Comment:
The values here are pretty standardized for now but we might want to support customization later.

Reply:
Yeah, seems good enough for starters.